CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing Guide
 
-*First*: if you feel insecure about how to start contributing, feel free to ask us on our [Discord channel](https://discordapp.com/invite/hasura), in the #contrib channel. You can also just go ahead with your contribution and we'll give you feedback. Don't worry - the worst that can happen is that you'll be politely asked to change something. We appreciate any contributions, and we don't want a wall of rules to stand in the way of that. 
+**: If you feel insecure about how to start contributing, feel free to ask us on our [Discord channel](https://discordapp.com/invite/hasura), in the #contrib channel**. You can also just go ahead with your contribution and we'll give you feedback. Don't worry - the worst that can happen is that you'll be politely asked to change something. We appreciate any contributions, and we don't want a wall of rules to stand in the way of that. 
```

Comment:
Let's remove only the first and keep the rest of the text in regular font weight.

Reply:
Okay, so how do I do it? I mean, I am new to open source and github, in general and I don't know how to edit a commit. Do I open a new PR now?

Reply:
Hope you figured out that you can keep pushing to the same PR :)